pkg/component/observability/plutono/dashboards/seed/fluent-bit-dashboard.json

```diff
@@ -1279,11 +1879,77 @@
         "tagsQuery": "",
         "type": "query",
         "useTags": false
+      },
+      {
+        "allValue": ".+",
+        "current": {
+          "selected": false,
+          "text": "All",
+          "value": "$__all"
+        },
+        "datasource": "prometheus-aggregate",
+        "definition": "label_values(fluentbit_vali_gardener_incoming_logs_total,host)",
+        "description": "Output plugin host target",
+        "error": null,
+        "hide": 0,
+        "includeAll": true,
+        "label": "Host",
+        "multi": true,
+        "name": "host",
+        "options": [],
+        "query": {
+          "query": "label_values(fluentbit_vali_gardener_incoming_logs_total,host)",
+          "refId": "StandardVariableQuery"
+        },
+        "refresh": 2,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "tagValuesQuery": "",
+        "tags": [],
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      },
+      {
+        "allValue": ".+",
+        "current": {
+          "selected": true,
+          "text": [
+            "All"
+          ],
+          "value": [
+            "$__all"
+          ]
+        },
+        "datasource": "prometheus-aggregate",
+        "definition": "label_values(valitail_sent_entries_total, host)",
+        "description": null,
+        "error": null,
+        "hide": 2,
+        "includeAll": true,
+        "label": "Url",
+        "multi": true,
+        "name": "url",
+        "options": [],
+        "query": {
+          "query": "label_values(valitail_sent_entries_total, host)",
+          "refId": "StandardVariableQuery"
+        },
+        "refresh": 2,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "tagValuesQuery": "",
+        "tags": [],
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
       }
     ]
   },
   "time": {
-    "from": "now-5m",
+    "from": "now-30m",
```

Comment:
Could you align the time and timepicker options with what's suggested in https://github.com/gardener/gardener/blob/d7bb834ac93f5eb18cebf2e7ae9e86653b5e826f/docs/development/monitoring-stack.md#L222-L248?